src/components/PeoplePage.tsx

```diff
@@ -1,29 +1,47 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useContext } from 'react';
+import { ErrorContext } from '../contsxts/ErrorContext';
+import { LoaderContext } from '../contsxts/LoaderContext';
```

Comment:
The import path for `LoaderContext` is incorrect. It should be '../contexts/LoaderContext' instead of '../contsxts/LoaderContext'.